wtpython/__main__.py

```diff
@@ -69,8 +69,8 @@ def main() -> None:
         print(e)
         return
 
+    traceback.print_exception(type(exc), exc, exc.__traceback__)
```

Comment:
probably an enhancement has an easy fix - use `rich`'s print to print the traceback. just needs a `print(''.join(traceback.format_exception(...)))` I believe